hack/install.sh

```diff
@@ -78,6 +78,10 @@ echo_docker_as_nonroot() {
 
 	Remember that you will have to log out and back in for this to take effect!
 
+	WARNING: adding non-root user to "docker" group, by default, will grant the
+	         ability to run containers which can be used to obtain root
+	         privileges on the docker host.
```

Comment:
Perhaps we should provide more information as to what this entails. "non-root" user may be a bit superfluous. How about;

```
WARNING: Adding a user to the "docker" group will grant the ability to run
         containers which can be used to obtain root privileges on the docker host.
         Refer to https://docs.docker.com/engine/security/security/#docker-daemon-attack-surface
         for more information.
```